contract/.env.example

```diff
@@ -1,2 +0,0 @@
-GOERLI_URL=""
```

Comment:
I think should have example let other one know which parm we should put into `.env`

so for example, should change to 

```
ROPSTEN_URL=""
...
```